src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || c >= a + b) {
+      throw new Error('Incorrect data');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const s: number = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(
+      s * ((s - this.a) * (s - this.b) * (s - this.c))
+    );
 
+    return Math.floor((area * 100) / 100);
```

Comment:
```suggestion
    return Math.floor(area * 100) / 100;
```

Reply:
check count getArea for triangle. looks like incorrect. should be to digits after comma